sql/moz-fx-data-marketing-prod/ga_derived/www_site_metrics_summary_v2/query.sql

```diff
@@ -0,0 +1,71 @@
+-- Query for ga_derived.www_site_metrics_summary_v2
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+WITH site_data AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', event_date) AS date,
+    device.category AS device_category,
+    device.operating_system AS operating_system,
+    device.web_info.browser AS browser,
+    device.language AS language,
+    geo.country AS country,
+    traffic_source.source AS source,
+    traffic_source.medium AS medium,
+    collected_traffic_source.manual_campaign_name AS campaign,
+    collected_traffic_source.manual_content AS ad_content,
+    SUM(CASE WHEN event_name = 'session_start' THEN 1 ELSE 0 END) AS sessions,
```

Comment:
Any reason not to simplify with this?
```suggestion
    COUNTIF(event_name = 'session_start') AS sessions,
```